synapse/config/workers.py

```diff
@@ -358,9 +358,9 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
                 "Must only specify one instance to handle `account_data` messages."
             )
 
-        if len(self.writers.receipts) != 1:
+        if len(self.writers.receipts) == 0:
             raise ConfigError(
-                "Must only specify one instance to handle `receipts` messages."
+                "Must specify at least one instance to handle `receipts` messages."
```

Comment:
We should update the documentation for this?

Reply:
I'm somewhat inclined to leave this undocumented until we can test it? But I could also be persuaded otherwise.

Reply:
That's fine. 👍